pkgs/development/python-modules/osqp/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildPythonPackage, fetchFromGitHub, lib, numpy, scipy, future, cmake }:
+
+buildPythonPackage rec {
+  pname = "osqp-python";
+  version = "2.0a1";
+
+  nativeBuildInputs = [ cmake ];
+
+  propagatedBuildInputs = [ numpy future scipy ];
+
+  src = fetchFromGitHub {
+    owner = "oxfordcontrol";
+    repo = "osqp-python";
+    rev = "v0.5.0";
+    fetchSubmodules = true;
```

Comment:
Do we really need this? Can we not package that one separately?

Reply:
I could download the source separately but AFAICT the python module needs the osqp source, i.e. it's not just linking against a library [1].

https://github.com/oxfordcontrol/osqp-python/blob/master/setup.py#L95